lib/smart_proxy_pulp_plugin/disk_usage.rb

```diff
@@ -67,5 +67,15 @@ def normalize_values(values)
       end
       values
     end
+
+    def validate_path(path_hash)
+      path_hash.each do |key, value|
+        unless File.readable?(value)
+          logger.debug "File at #{value} defined in #{key} parameter doesn't exist or is unreadable"
```

Comment:
I'd suggest using a warning level here instead -- debug isn't normally used in production and configuration errors may end up being hidden because of that.